src/deepsparse/pipeline.py

```diff
@@ -703,6 +703,7 @@ def log(self, identifier: str, value: Any, category: str):
         :param value: The logged data structure
         :param category: The metric category that the log belongs to
         """
+        validate_identifier(identifier)
         if self.logger:
             self.logger.log(
                 identifier=f"{self.alias or self.task}.{identifier}",
```

Comment:
Isn't this an immediate violation?

Reply:
should change the `.` here before `{identifier}` to some thing else as discussed offline. final identifier should pass validation as well